pages/22.What's New/docs.en.md

```diff
@@ -13,6 +13,32 @@ taxonomy:
 ----------------------------
 In this section we'll highlight the new features for each major release of Mautic.  They are in version order beginning with the latest.
 
+##Version 3.0
+Mautic 3.0 introduces significant features and a number of enhancements. They are outlined as follows:
+
+###Hosting Requirements
+- Mautic 3.0 added support for PHP versions 7.2 and 7.3. The minimum PHP version is raised from version 5.6.19 to version 7.2.0.
+- The minimum MySQL version is increased from version 5.5.3 to 5.7.14.
+
+###Cron Jobs Update
+There are significant changes in the file structure in Mautic 3. To comply with the change, you will need to update your cron jobs path.
+
+Example:
+
+`app/console mautic:segments:update`
+
+should now be:
+
+`bin/console mautic:segments:update`
+
+> Note: For Mautic version 2.0, you must continue using the `app/console` path for running any cron job.
+
+###Removed Plugins
+Mautic 3.0 removed support for the Rackspace and OpenStack plugins for remote assets.
+While uploading an asset on the New Asset page, if you select Remote Storage location and click Browse Remote Files, you will see only Amazon S3 as the available plugin. The Rackspace and OpenStack plugins are no longer available.
```

Comment:
They will only see these options if they have them configured, so I would perhaps suggest removing this as I don't think it would be likely for people to have all three plugins enabled and configured.  Generally people will use one storage provider in a Mautic instance.

```suggestion
If you currently use Rackspace or OpenStack you will need to switch to Amazon S3, or create custom plugins to support alternative providers.
```